.gitignore

```diff
@@ -29,6 +29,9 @@
 .vs/
 .vscode/
 
+# Visual studio project config
+.vsconfig
```

Comment:
I just added this in a commit on one of Ryan's PRs that was just merged. You can remove this file from your PR.

Reply:
Removed in latest push.